tfx_addons/example_filter/gcp.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Is this used?

Reply:
Would suggest instead to write one without GCP which just runs locally. There's a lot of boilderplate here which is not needed